script/Ciel/Day 10/10_01_CIEL10_4_1.txt

```diff
@@ -186,16 +206,20 @@ A frightening sound of astonishment that I would never have expected to hear fro
 [sha:5ea6c7d9e2480693d82724dfbd36b42f02be4951]{
 -- Page 183, Offset 27558.
 -- 「……25歳って……潜入任務ですから、多少の偽装はするでしょうけど……シスター・ノエル……」
-"Twenty-five years old, she says... An undercover operation needs some degree of disguise, but... Sister Noel..."
+"Twenty-five years old, she says...
+ I appreciate that an undercover mission calls for a disguise, but... Sister Noel..."
```

Comment:
I think the tone should be a bit more.. mocking?
>Really, twenty-five years old...
>I mean, given, undercover missions do call for some level of disguise, but... Sister Noel...

Reply:
VA kinda sounds more exasperated to me here than mocking

Reply:
I mean, that's why it's just a bit more.